dev-build/cmake/cmake_gui-3.30.3.recipe

```diff
@@ -0,0 +1,116 @@
+SUMMARY="A cross-platform build system (GUI version)"
+DESCRIPTION="CMake is a family of tools designed to build, test and package \
+software. It is used to control the software compilation process using \
+simple platform and compiler independent configuration files. CMake generates \
+native makefiles and workspaces that can be used in the compiler environment \
+of your choice.
+
+This package provides the GUI version of CMake (cmake-gui)"
+HOMEPAGE="https://cmake.org/"
+COPYRIGHT="2002-2024 Kitware, Inc., Insight Consortium"
+LICENSE="CMake"
+REVISION="1"
+SOURCE_URI="https://cmake.org/files/v${portVersion%.*}/cmake-$portVersion.tar.gz"
+CHECKSUM_SHA256="6d5de15b6715091df7f5441007425264bdd477809f80333fdf95f846aaff88e4"
+SOURCE_DIR="cmake-$portVersion"
+PATCHES="cmake-$portVersion.patchset"
+ADDITIONAL_FILES="cmake.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+
+PROVIDES="
+	cmake_gui$secondaryArchSuffix = $portVersion
+	cmd:cmake_gui = $portVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmake$secondaryArchSuffix >= 3	# for the dataDir/cmake files.
+	lib:libarchive$secondaryArchSuffix
+	lib:libcppdap$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libjsoncpp$secondaryArchSuffix
+	lib:libQt6Core$secondaryArchSuffix
+	lib:libQt6Gui$secondaryArchSuffix
+	lib:libQt6Widgets$secondaryArchSuffix
+	lib:librhash$secondaryArchSuffix
+	lib:libstdc++$secondaryArchSuffix
+	lib:libuv$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
```

Comment:
Maybe add

```sh
if [ -n "$secondaryArchSuffix ]; then
	REPLACES="cmake${secondaryArchSuffix}_gui"
fi
```
to facilitate upgrades from an older version that still had the GUI.